pootle/static/js/editor/app.js

```diff
@@ -2091,8 +2094,17 @@ PTL.editor = {
         $(data.timeline).hide().prependTo('#extras-container')
                         .slideDown(1000, 'easeOutQuad');
       }
-
-      helpers.updateRelativeDates();
+      $('.js-mount-timesince').each((index, el) => {
```

Comment:
Please do not use jQuery in new code.
```js
[...document.querySelectorAll('.js-mount-timesince')].forEach((el, i) => {
  // do stuff
});
```

_(Note to self: we should probably have a wrapper for that so that `Q('.selector')` works transparently, with less keystrokes and no need to spread manually)._